README.md

```diff
@@ -61,20 +49,38 @@ Show work-in-progress areas in interactive mode:
 ./compile.sh EXPERT="yes"
 ```
 
-Build minimal CLI Armbian Jammy for Raspberry Pi 4B with LTS kernel and write image directly to the SD card:
+Build minimal CLI Armbian Jammy for UEFI x86 with LTS kernel:
```

Comment:
Why on earth should we promote x86? This makes no sense to me...

Reply:
Changed to generic arm64 ... reason is that x86 compilation is (currently) significantly faster and rpi builds is currently broken.

Reply:
Why not using something platinum supported or very popular? Like VIM4 or OPi5+?

Reply:
Orangepi zero was previously in as one of most popular. vim4 and opi5 are in the legacy / WIP world. Not sure if this here as some significance.

Reply:
Change to Bananapi M5.